icons/planet.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="8" />
+  <path d="M4.05 13c-1.7 1.8-2.5 3.5-1.8 4.5 1.1 1.9 6.4 1 11.8-2s8.9-7.1 7.7-9c-.6-1-2.4-1.2-4.7-.7" />
```

Comment:
This one feels a bit small, I think we should put the ring at a 45deg angle so that more space is available to let it grow a bit. This would also fix the 2px gap guideline violation:

![image](https://github.com/lucide-icons/lucide/assets/17746067/f69437d3-2382-405b-a6fa-efe6e0127e18)

```suggestion
  <circle cx="12" cy="12" r="9" />
  <path d="M14.93 3.49A13.23 4.99 135 1 1 3.49 14.93" />
```